components/raftstore/src/store/fsm/peer.rs

```diff
@@ -710,12 +716,20 @@ where
             }
         }
         self.on_loop_finished();
+        let elapsed = timer.saturating_elapsed();
+        slow_log!(
```

Comment:
Using `slow_log` is a better choice ? Why not use `metrics` ?

The default threshold for slow logging is `1s`. IMO, it's not appropriate for logging slow `PeerMsg` by `slow_log`.

Reply:
We have to get the duration for each message if using metrics, which may have some overhead in the hot path.

Reply:
Let me change the slow threshold for this

Reply:
The output of distribution ought to be something like this [x, y, z, ...,], maybe we can wrap it and implements a fmt to make it read friendly?

Reply:
How about adding to metrics if an event exceeds 100ms?